html/modules/custom/iasc_content/src/Controller/GroupMeetings.php

```diff
@@ -125,7 +125,16 @@ public function getEvents(Request $request, Group $group) : array {
     $index = Index::load('default_solr_index');
 
     // Render array.
-    $build = [];
+    $build = [
```

Comment:
@attiks I didn't need the main wrapper, the individual wrappers were better. So this can be removed if you wish.